Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.2
+MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com
+
+WORKDIR /gopath/src/github.com/google/cadvisor
+COPY    . /gopath/src/github.com/google/cadvisor
+
+RUN apk add --update --repository=http://dl-4.alpinelinux.org/alpine/edge/community/ -t build-deps go sqlite-dev btrfs-progs-dev bash linux-headers make git gcc g++ && \
+    apk add --update device-mapper && \
+    export GOPATH=/gopath && \
```

Comment:
Yeah if none of the tools are present, it doesn't help much.  I was thinking in the future we could even support running tests through Docker to make cross platform development easier, but this isn't a goal at this point.